controllers/machinehealthcheck_controller.go

```diff
@@ -250,12 +250,17 @@ func (r *MachineHealthCheckReconciler) reconcile(ctx context.Context, logger log
 	// mark for remediation
 	errList := []error{}
 	for _, t := range unhealthy {
+		if annotations.HasPausedAnnotation(t.Machine) {
```

Comment:
This could probably use a comment as to why it's not using `annotations.IsPaused` (primarily because it's called in `Reconcile` and if cluster.Spec.Paused was true we wouldn't make it here).

Reply:
I just switched it to use `IsPaused`, that seems simpler than adding a comment